lib/fcmpush/client.rb

```diff
@@ -26,7 +26,7 @@ def initialize(domain, project_id, configuration, **options)
     def v1_authorize
       @auth ||= if configuration.json_key_io
                   Google::Auth::ServiceAccountCredentials.make_creds(
-                    json_key_io: File.open(configuration.json_key_io),
+                    json_key_io: configuration.json_key_io,
                     scope: configuration.scope
                   )
                 else
```

Comment:
I want to keep backward compatibility.
I think this is better. Isn't it?

```suggestion
      @auth ||= if configuration.json_key_io
	              io = if configuration.json_key_io.is_a?(IO)
		                 configuration.json_key_io
		               else
		                 File.open(configuration.json_key_io)
	                   end
                  Google::Auth::ServiceAccountCredentials.make_creds(
		            json_key_io: io,
		            scope: configuration.scope
		          )
                else
                  # from ENV
                  Google::Auth::ServiceAccountCredentials.make_creds(scope: configuration.scope)
                end
```

Reply:
I want you to write some test.
Could you do it?

Reply:
oh...
I realized that test needs firebase project and credentials..
Don't care about test!

Reply:
Yes, it is!
I'll fix it.

Reply:
I fixed it by 644a000.
Could you please check again?

Reply:
Quick fix!
Thanks!